source/catalog.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Мишка</title>
+</head>
+
+<body>
+
+  <header class="page-header">
+    <a class="page-header__logo" href="index.html">
+      <img class="page-header__logo-image" src="https://via.placeholder.com/86x35" width="86" height="35"
+        alt="Логотип магазина Мишка">
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__open-button" type="button">Открыть меню</button>
+      <ul class="main-nav__list cite-list">
+        <li class="cite-list__item">
+          <a class="cite-list__link">Каталог товаров</a>
+        </li>
+        <li class="cite-list__item">
+          <a class="cite-list__link" href="form.html">Вязание на заказ</a>
+        </li>
+      </ul>
+      <ul class="main-nav__list cite-list cite-list--user">
+        <li class="cite-list__item">
+          <a class="cite-list__link cite-list__link--search" href="#">Поиск по сайту</a>
+        </li>
+        <li class="cite-list__item">
+          <a class="cite-list__link cite-list__link--cart" href="#">Корзина: пока пуста</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+
+  <main class="page-main">
+    <h1 class="page-main__title">Каталог товаров</h1>
+    <section class="catalog">
+      <h2 class="visually-hidden">Каталог-товаров</h2>
+      <ul class="catalog__list catalig-list">
```

Comment:
```suggestion
      <ul class="catalog__list catalog-list">
```

https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker-russian